File: src/Subgraph_isomorphism.R
#' K nearest neighbours algorithm
#'
#' @param pattern.mat Graf v katerem iščemo vzorec
#' @param match.mat za ujemanje, vzorec 
#' @param method preiskovanja
#' @param exit_type Povemo kakšen tip izhoda smo si poželeli
#'
#' @return
#' @export
#'
#' @examples
Subgraph_isomorphism <- function(pattern.mat,match.mat,method,exit_type){
  result.list  <- .C("Subgraph_isomorphism_interface",
                     as.double(pattern.mat),
                     as.double(match.mat),
                     method,
                     exit_type,
                     output = "",
                     PACKAGE = "Sics")
  
  result.list$output
  #x.mat
}

File: src/Subgraph_isomorphism.cpp
#include <cstdint>
#include <fstream>
#include <iostream>
#include <string> 
#include <stdio.h>

#include <string.h>

using namespace std;

int Subgraph_isomorphism(
    const double *pattern, //n_observations x n_features
    const double *match,   //n_observations
    const char *method,
    const char *exit_type,
    char **output
  ){
  std::string pristevanka = "..";
  int i, j;
  for (i = 0; i < 3; i++){
    for (j = 0; j < 3; j++){
      pristevanka = pristevanka + "dida";
    }
  }
  

  strcpy(*output, pristevanka.c_str());

    return 0;
  }
















